### PR TITLE
Update cla.yml and tests.yml

### PR DESCRIPTION
**Pull request by sverdlov93**
- [ ] This pull request is created in
  the [jfrog/jenkins-artifactory-plugin](https://github.com/jfrog/jenkins-artifactory-plugin) repository.
- [ ] All [tests](https://ci.appveyor.com/project/jfrog-ecosystem/jenkins-artifactory-plugin) passed. If this feature is
  not already covered by the tests, I added new tests.
- [ ] All [static analysis checks](https://github.com/jfrog/jenkins-artifactory-plugin/actions/workflows/analysis.yml)
  passed.
-----

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -46,15 +46,15 @@ jobs:
...
+          nuget-version: 6.x
 
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
           python-version: "3.x"
 
       - name: Setup Python virtual environment
-        run: pip install virtualenv && python -m virtualenv venv
+        run: python -m pip install virtualenv && python -m virtualenv venv
```
Comment (eyalbe4) [inline .github/workflows/tests.yml:57]: It looks like this workflow oesn't need python or pip, so we can remove this.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -46,15 +46,15 @@ jobs:
       - name: Install NuGet
         uses: nuget/setup-nuget@v1
         with:
-          nuget-version: 5.x
+          nuget-version: 6.x
 
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
           python-version: "3.x"
```
Comment (eyalbe4) [inline .github/workflows/tests.yml:54]: Let's remove this python setup, since it doesn't seem to be used,

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -46,15 +46,15 @@ jobs:
       - name: Install NuGet
         uses: nuget/setup-nuget@v1
         with:
-          nuget-version: 5.x
+          nuget-version: 6.x
```
Comment (eyalbe4) [inline .github/workflows/tests.yml:49]: Let's remove, because it isn't used..

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.17.x
+          go-version: 1.20.x
```
Comment (eyalbe4) [inline .github/workflows/tests.yml:33]: Let's remove, because it isn't used..

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (sverdlov93) [reply .github/workflows/tests.yml:57]: The tests here use python

**Pull request merged**